tests/security/oscap_pci_dss_4/oscap_xccdf_eval_remote.pm

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'pci-dss' hardening in the 'scap-security-guide': detection mode with remote
+# Maintainer: QE Security <none@suse.de>
+# Tags: poo#93886, poo#104943
+
+use base 'oscap_tests';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils qw(is_sle);
```

Comment:
please double-check for libraries not used in all files (unfortunately Perl or the IDE doesn't help us much here).

Reply:
Checked